src/panels/config/energy/components/ha-energy-validation-result.ts

```diff
@@ -31,7 +31,10 @@ class EnergyValidationMessage extends LitElement {
             }
           >
             ${this.hass.localize(
-              `ui.panel.config.energy.validation.issues.${issueType}.description`
+              `ui.panel.config.energy.validation.issues.${issueType}.description`,
+              issueType === "entity_unexpected_unit_price"
+                ? { currency: this.hass.config.currency }
```

Comment:
Should we use a "currency" placeholder here?
`expected units of measurement "/kWh" or "/Wh"` is misleading as well?

Reply:
The currency will always be defined?

Reply:
Right!👍